src/openmc_driver.cpp

```diff
@@ -21,6 +20,31 @@ OpenmcDriver::OpenmcDriver(MPI_Comm comm)
     err_chk(openmc_init(0, nullptr, &comm));
   }
   MPI_Barrier(MPI_COMM_WORLD);
+
+  // determine number of fissionable cells in model to aid in catching
+  // improperly mapped problems
+  n_fissionable_cells_ = 0;
+  for (int i = 0; i < cells_size(); ++i) {
```

Comment:
Maybe `cells_.size()` ?

Reply:
Also use `gsl::index`

Reply:
It should be `cells_size`, it's from the OpenMC API

Reply:
So I see this function defined in `openmc/src/cell.cpp`:
```
extern "C" int cells_size() { return model::cells.size(); }
```

But I don't see it declared in `capi.h`, nor in any other header.  That must be why Travis is failing with this error:
```
/home/travis/build/enrico-dev/enrico/src/openmc_driver.cpp: In constructor ‘enrico::OpenmcDriver::OpenmcDriver(MPI_Comm)’:

/home/travis/build/enrico-dev/enrico/src/openmc_driver.cpp:27:34: error: ‘cells_size’ was not declared in this scope

   for (int i = 0; i < cells_size(); ++i) {

                                  ^

CMakeFiles/libenrico.dir/build.make:114: recipe for target 'CMakeFiles/libenrico.dir/src/openmc_driver.cpp.o' failed
```

Reply:
yep :) in the process of fixing that!

[(https://github.com/openmc-dev/openmc/pull/1310)](url)

Reply:
weird, not sure why that link doesn't work... anyways, I have an MR open for it

Reply:
\*wonders how long it will be until April starts saying PR, not MR\* :smile:

Reply:
Dude, I think MR is way better.  The term "pull request" has a confusing frame of reference.  Seems like it could also be called a "push request", right?  The term "merge request" avoids all that nonsense.

Reply:
Ha, I use the two pretty interchangeably. And the notifications I get through gitlab when something is merged is "merge request merged." So I didn't make it up :)